legacy/config/config.d.ts

```diff
@@ -62,3 +62,11 @@ type WalletConfig = {
 }
 
 export var getWalletConfigById: (walletImplementationId: string) => WalletImplementation
+
+// prettier-ignore
+export var getCardanoBaseConfig: (networkConfig: CardanoHaskellShelleyNetwork) => Array<{
+  StartAt?: number,
+  GenesisDate?: string,
+  SlotsPerEpoch?: number,
+  SlotDuration?: number,
```

Comment:
```suggestion
  startAt?: number,
  genesisDate?: string,
  slotsPerEpoch?: number,
  slotDuration?: number
```
properties on camelCase